### PR TITLE
this commit add GET /api/articles/:article_id

### PR DESCRIPTION
**Pull request by rsr83**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -45,7 +45,50 @@ describe("/api", () => {
...
   });
 });
+
+describe("/api/articles/:articles_id", () => {
+  test("GET 200: receive a the article requested by their respective id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js:60]: Well done, this is a really important thing to check and make sure the article id is what you want it to be!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -45,7 +45,50 @@ describe("/api", () => {
...
+        expect(response.body.article.title).toBe(
+          "Living in the shadow of a great man"
+        );
+        expect(response.body.article.topic).toBe("mitch");
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js:64]: .toMatchObject could be a better matcher to use here to make your code more _DRY_

File: __tests__/app.test.js [hunk 3]
```diff
@@ -45,7 +45,50 @@ describe("/api", () => {
...
+  });
+  test("GET:404 sends an appropriate status and error message when given a valid but non-existent id", () => {
+    return request(app)
+      .get("/api/articles/999")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Article not found");
+      });
+  });
+  test("GET:400 sends an appropriate status and error message when given an invalid id", () => {
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js:86]: Great error tests here👌

**Review by nc-beth-mccormack — COMMENTED** [review]
Nice work well done!

File: __tests__/app.test.js (hunk 2 above)
Comment (rsr83) [reply __tests__/app.test.js:64]: so I should check for the Id only and the for the others properties I dont need to check one by one?

**Pull request merged**